### PR TITLE
feat(vue3): typescript

### PR DESCRIPTION
**Pull request by DSergiu**
Closing #50 and #65.

* adds typescript support to vue3 sdk
* sync versions on `1.2.0`
* adds CHANGES.md to track changes

### CODE REVIEW

File: examples/ts-vue3/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+This template should help get you started developing with Vue 3 in Vite.
+
+## Customize configuration
+
+See [Vite Configuration Reference](https://vitejs.dev/config/).
+
+## Project Setup
+
+```sh
+npm install
```
Comment (zoryana94) [inline examples/ts-vue3/README.md:12]: what about using yarn here as well? just for consistency

File: examples/ts-vue3/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,158 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import VueHcaptcha from '@hcaptcha/vue3-hcaptcha';
```
Comment (zoryana94) [inline examples/ts-vue3/src/App.vue:3]: just for the consistency, let's use one type of quotes here: singular or double

File: packages/vue3/package.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
   "files": [
     "dist"
   ],
+  "types": "types/index.d.ts",
```
Comment (zoryana94) [inline packages/vue3/package.json:25]: I don't see it exists

File: examples/ts-vue2/README.md [hunk 4]
```diff
@@ -4,10 +4,10 @@ Demo app to showcase usage of the vue web component in a ts class-style vue app.
...
 ## Project setup
 ```
-npm install
+yarn install
 ```
 
 ### Run demo
 ```
-npm run serve
+yarn serve
```
Comment (zoryana94) [inline examples/ts-vue2/README.md:12]: it's working well, however, there are some type issues

**Review by zoryana94 — APPROVED** [review]

File: packages/vue3/package.json (hunk 3 above)
Comment (DSergiu) [reply packages/vue3/package.json:25]: The pipeline will copy the types from `packages/vue2` before publishing.

File: examples/ts-vue3/README.md (hunk 1 above)
Comment (DSergiu) [reply examples/ts-vue3/README.md]: Agree, fixed https://github.com/hCaptcha/vue-hcaptcha/pull/86/commits/b68905105c83328d4f25a18c2741a3eb7a0891d5

File: examples/ts-vue3/src/App.vue (hunk 2 above)
Comment (DSergiu) [reply examples/ts-vue3/src/App.vue]: Fixed https://github.com/hCaptcha/vue-hcaptcha/pull/86/commits/b68905105c83328d4f25a18c2741a3eb7a0891d5

File: examples/ts-vue2/README.md (hunk 4 above)
Comment (DSergiu) [reply examples/ts-vue2/README.md:12]: Not a priority to fix this atm.

**Review by zoryana94 — APPROVED** [review]

**Pull request merged**